tc/sgx/trusted_worker_manager/enclave/work_order_enclave.cpp

```diff
@@ -16,31 +16,19 @@
 #include "enclave_t.h"
 
 #include <string>
-#include <vector>
 
 #include <sgx_trts.h>
-#include <sgx_tseal.h>
-#include <sgx_utils.h>
-#include "sgx_thread.h"
 #include<mbusafecrt.h>
 
 #include "error.h"
-#include "packages/base64/base64.h"
 #include "tcf_error.h"
-#include "timer.h"
-#include "types.h"
-#include "zero.h"
 
 #include "enclave_utils.h"
-
 #include "base_enclave.h"
-#include "work_order_enclave.h"
```

Comment:
Why `work_order_enclave.h` is not included in `work_order_enclave.cpp`

Reply:
Its not needed, as the function signatures(enclave_t.h) will be generated by sgx_edger tool.